.changeset/early-feet-lay.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-auth-backend-module-oidc-provider': minor
+---
+
+Added custom timeout setting for oidc provider
```

Comment:
Could you please add an example here on how the custom config can be used in the `app-config.yaml` file?

Reply:
Hi, thank you for your feedback. I have added an example.